frontend/language/src/en.json

```
@@ -631,6 +631,10 @@
   "schema_editor.info_dialog_title": "New data models page",
   "schema_editor.integer": "Integer",
   "schema_editor.keyword": "Keyword",
+  "schema_editor.language_add_language":"Add language",
+  "schema_editor.language_conferm_deletion": "Yes, delete language",
+  "schema_editor.language_display_confirm_delete": "Are you sure you want to delete this language?",
+  "schema_editor.language_info_melding": "We recommend adding translations for Bokmål, Nynorsk and English. If necessary, you can also add other languages.",
```

`melding` should be translated to `message`:
```suggestion
  "schema_editor.language_info_message": "We recommend adding translations for Bokmål, Nynorsk and English. If necessary, you can also add other languages.",
```

---

fixed